base/random.jl

```diff
@@ -1194,6 +1196,9 @@ The `Base` module currently provides an implementation for the types
     end
 end
 
+Base.@irrational SQRT_HALF 0.7071067811865475244008  sqrt(big(0.5))
+randn{T}(rng::AbstractRNG, ::Type{Complex{T}}) = Complex{T}(SQRT_HALF * randn(rng, T), SQRT_HALF*randn(rng, T))
```

Comment:
I think it would make an `InexactError` when calling the complex constructor on real (because of the multiplication by `SQRT_HALF`) values, e.g. with `T==Int`. Also, this distribution will make sense only when `randn(T)` produces a number in `[0,1)`, i.e. `T<:AbstractFloat`.